Fortnox.NET.Tests/Fortnox.NET.Tests/AssetsTest.cs

```diff
@@ -185,7 +185,7 @@ public async Task ItCanDepreciateAnAsset()
             var depreciatedAssetResponse = await AssetsService.DepreciateAssetAsync(request, assetDepreciation);
 
             var depreciationResult = depreciatedAssetResponse.Data.First();
-            Assert.IsTrue(depreciationResult.FinancialYear == 2);
+            Assert.IsTrue(depreciationResult.FinancialYear == 3);
```

Comment:
I´m a bit worried about the variables in these tests changing ever so often. Are these tests flaky or dependant on the API? If so, can we rework them somehow? It's problematic having to rely on some iterating or changing variable. I rember having to reset tests by deleting recently added data etc, maybe we can do that if need be?

Reply:
does this require a patch version bump btw?

Reply:
This does not require any version bumps it simply fixes some tests that failed. This specific test failed due to the assets being set to depreciate in the next financial year due to it now being december month 12. 

Do you think we should look over theese tests again?

Reply:
as we spoke about in person, we should look over these, but some things are probably dependant on the behaviour of Fortnox API and out of our control, as in flaky by nature. But let's do this over time as issues surface. If possible to change any of the tests affected by the PR to avoid having to manually adjut them, please do it.